<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationStorage</class>
 <widget class="QWidget" name="StationStorage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>站存车信息</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>631</width>
     <height>38</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>车站：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>修改时间：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>发送时间：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>接收时间：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>631</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,2">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_7" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_8" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>210</y>
     <width>631</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,4">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_5" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>股道存车信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_9" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_6" native="true">
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>18</y>
         <width>24</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>类别</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>53</y>
         <width>24</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>量数</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>18</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>53</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>18</y>
         <width>24</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>车种</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>18</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>18</y>
         <width>48</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>作业说明</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_5">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>18</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>53</y>
         <width>48</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>运用标记</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_6">
       <property name="geometry">
        <rect>
         <x>177</x>
         <y>53</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>88</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>说明/去向</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>79</x>
         <y>88</y>
         <width>251</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>123</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>插入</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>123</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>123</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>123</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="horizontalSpacing">
     <number>21</number>
    </property>
    <property name="verticalSpacing">
     <number>3</number>
    </property>
    <item row="0" column="4">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>上报调度所</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_GDEdit">
      <property name="text">
       <string>编辑股道</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>合计车数:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
